File: Java/Laba8/src/main/java/ua/lviv/iot/goods/goods/Calendar.java
package ua.lviv.iot.goods.goods;

import ua.lviv.iot.goods.enums.CategoryEnum;

public class Calendar extends Good {
    private String year;

    public Calendar(final Integer id, String name, CategoryEnum categoryEnum, boolean aviability, int price, String year, int pages) {
        super(id, name, categoryEnum, aviability, price, 1);
        this.year = year;
    }

    public String getYear() {
        return year;
    }

    public void setYear(String year) {
        this.year = year;
    }

    @Override
    public String getHeaders() {
        return super.getHeaders() + "year";
    }

    @Override
    public String toCSV() {
        return super.toCSV() + year;
    }
}

File: Python/lab13/goods/Calendar.py
from goods.Good import Good


class Calendar(Good):
    def __init__(self, name, category, price, year, pages):
        super().__init__(name, category, price, year, pages)
        self.year = year

    def __str__(self):
        return ", Name:" + str(self.name) + " Category: " + str(self.category) + ", Price:" + str(
            self.price) + ", Year:" + str(self.year) + ", Pages:" + str(self.pages)

